asset/js/setup.js

```diff
@@ -33,10 +33,17 @@ function setupAnchors() {
   });
 }
 
+let userAfterSetup = () => {};
+// eslint-disable-next-line no-unused-vars
+function afterSetup(func) {
+  userAfterSetup = func;
```

Comment:
> * This means I can only have one `afterSetup` that works?
> * Is there a race condition? If I call `afterSetup` after setup is done (e.g. in console), then `func` should still run. Can refer to how jQuery implements [`$(handler)`](https://api.jquery.com/ready/).

Yes and yes. I did not anticipate the multiple call case, but no reason we shouldn't make that work. Seems like it would be easier to have our function call `jQuery(handler)` instead, since we are already using it anyway.

> * Shall we namespace (and rename) this as `MarkBind.ready(func)`?

Sure, this makes more sense if we have to implement more functions of this nature.

Reply:
> Seems like it would be easier to have our function call `jQuery(handler)` instead, since we are already using it anyway.

That would execute it after jQuery is ready, rather than after Vue is mounted.

Reply:
I have implemented it with [`jQuery.Deferred()`](https://api.jquery.com/jquery.deferred/), please let me know if that's okay!